### PR TITLE
remove wrapping of inline comments in <ul>

### PR DESCRIPTION
**Pull request by gregeinfrank**
This was causing parsing issues for inline html within an inline comment - and added unnecessary complexity for not much benefit. This removes the `<ul>` list of inline comments for a GH comment and separates them out by newlines instead.

fixes: https://app.asana.com/0/1149418478823393/1201442536592555/f

Pull Request synchronized with [Asana task](https://app.asana.com/0/0/1202038153443517)

### CODE REVIEW

File: src/markdown_parser.py [hunk 1]
```diff
@@ -31,6 +31,9 @@ def inline_html(self, html) -> str:
     def block_html(self, html) -> str:
         return escape(html)
 
+    def linebreak(self) -> str:
```
Comment (ericwalton-asana) [inline src/markdown_parser.py:34]: Looks like this is unused. Did you mean to call it in line 358 above?

File: src/asana/helpers.py [hunk 2]
```diff
@@ -350,14 +350,12 @@ def asana_comment_from_github_review(review: Review) -> str:
...
-            _wrap_in_tag("A", attrs={"href": comment.url()})(f"[{i}] ")
-            + _format_github_text_for_asana(comment.body())
-        )
+        _wrap_in_tag("A", attrs={"href": comment.url()})(f"[{i}] ")
+        + _format_github_text_for_asana(comment.body())
         for i, comment in enumerate(review.comments(), start=1)
     ]
     if inline_comments:
-        comments_html = _wrap_in_tag("ul")("".join(inline_comments))
+        comments_html = "\n".join(inline_comments)
```
Comment (ericwalton-asana) [inline src/asana/helpers.py:358]: I think I'd recommend two new lines, so that there's a blank space between comments.

**Review by ericwalton-asana — CHANGES_REQUESTED** [review]
Two little nits. Seems very reasonable to do.

File: src/markdown_parser.py (hunk 1 above)
Comment (gregeinfrank) [reply src/markdown_parser.py:34]: This overrides the default from [mistune](https://github.com/lepture/mistune/blob/46a65a978b005651dcc54a340e44aef71f9f7e95/mistune/renderers.py#L163) and is used internally by that parsing library

File: src/asana/helpers.py (hunk 2 above)
Comment (gregeinfrank) [reply src/asana/helpers.py:358]: I originally had it that way, but when I went to test it, it ended up with 3 newlines (this ends up with 2). I think this has to do with the fact that each inline comment itself ends in a `"\n"`, so this essentially adds the blank space for us

**Comment by gregeinfrank** [comment]
I will update `black` version in a follow-up PR, there seems to be an issue running it in CI

**Review by ericwalton-asana — APPROVED** [review]
LGTM!

**Pull request merged**